client/jetpack-connect/main.jsx

```diff
@@ -348,7 +348,7 @@ class JetpackConnectMain extends Component {
 		const { translate } = this.props;
 		return (
 			<LoggedOutFormLinks>
-				<JetpackConnectHappychatButton>
+				<JetpackConnectHappychatButton eventName="calypso_jpc_firststep_chat_initiated">
```

Comment:
I'd argue the `firststep` is a little ambiguous. There are some flows where this won't actually be the first step. Perhaps we should name it `siteentry` or something similar instead?